server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/method/EmailQueryMethod.scala

```diff
@@ -114,6 +114,6 @@ class EmailQueryMethod @Inject() (serializer: EmailQuerySerializer,
     (request.anchor, request.anchorOffset) match {
       case (maybeAnchor, _) if maybeAnchor.isDefined => SMono.raiseError(UnsupportedRequestParameterException("anchor"))
       case (_, maybeAnchorOffset) if maybeAnchorOffset.isDefined => SMono.raiseError(UnsupportedRequestParameterException("anchorOffset"))
```

Comment:
```
case (Some(anchor), _) => ...
case (_, Some(anchorOffset)) => ...
```

Reply:
The S3 rebase made me forget all about Scala... :D